features/step_definitions/sign_up.rb

```diff
@@ -0,0 +1,80 @@
+require 'faker'
+
+# Navigation to sign up page:
+
+Given('I am on the login page') do
+  navigate_to(ENV['URL'])
+end
+
+And('I click on the registration button') do
+  click(:id, 'sign-up')
+end
+
+# Successful registration:
+
+When('I fill all the required fields') do
```

Comment:
before performing any action it is good to first validate that the elements exist 

#is_element_displayed(access_type, access_name) ⇒ Object
#is_element_enabled(access_type, access_name) ⇒ Object
#wait_for_element_to_display(access_type, access_name, duration) ⇒ Object
#wait_for_element_to_enable(access_type, access_name, duration) ⇒ Object